pyat/at/lattice/lattice_object.py

```diff
@@ -55,10 +55,13 @@ class Lattice(list):
     """
     # Attributes displayed:
     _disp_attributes = ('name', 'energy', 'particle', 'periodicity',
-                        'harmonic_number')
+                        'harmonic_number', 'beam_current', 'nbunch')
+    # excluded attributes
+    _excluded_attributes = ('nbunch', 'beam_current')
```

Comment:
I think that you should not exclude `beam_current`. It prevents initialising it with a keyword. It does not matter if `beam_current` is transmitted in save/load while `_fillpattern` is not, they are completely independent.

Reply:
Right, I thought that setting `bunch_current=[beam_current] `behind the users back when reloading a file was dangerous...while setting the current at initialization was not so critical.
I'll remove it if you think it is better the other way